dev/system-test/firestore.ts

```diff
@@ -5319,30 +5319,6 @@ describe('Transaction class', () => {
     expect(snapshot.exists).to.be.true;
     expect(snapshot.get('foo')).to.equal(1);
   });
-
```

Comment:
I added client side exceptions to attempting write on readonly transaction, so this integration test is no longer valid.